features/collections/collection_participants.feature

```diff
@@ -67,6 +67,20 @@
   Then I should see "You are now a member of Such a nice collection"
   When I am in the default browser
 
+  Scenario: A subcollection profile and blurb does not show duplicates when a moderator is also an owner of the parent collection
+    Given a user exists with login: "sam"
+      And I have the collection "Such a nice collection" with name "Collection"
+      And I am logged in as the owner of "Such a nice collection"
+      And I have added the co-owner "sam" to collection "Such a nice collection"
+      And I add the subcollection "Such a nice subcollection" to the parent collection named "Collection"
```

Comment:
This is very nitpicky, but I think it would be easier to follow what is going on if the collection titles weren't so long and so similar. Just "Subcollection" and "Collection" might work. (If you do that, you can also leave off the optional "with name" part of the first step. The way that step is set up, it will automatically have the name "Collection.")

Reply:
Sounds good!